libraries/chain/steem_evaluator.cpp

```diff
@@ -2358,20 +2422,11 @@ void claim_account_evaluator::do_apply( const claim_account_operation& o )
          uint32_t delta_time = ( _db.head_block_time() - current_witness.last_subsidy_update ).to_seconds();
 
          // linear decay to enforce no more than 10% max by a single witness over 48 hours.
-         uint64_t recovered_subsidies = ( delta_time * STEEM_100_PERCENT * wso.single_witness_subsidy_limit )
-            / ( fc::days( STEEM_ACCOUNT_SUBSIDY_BURST_DAYS ).to_seconds() * STEEM_100_PERCENT );
-         uint64_t new_subsidies = current_witness.recent_account_subsidies;
+         uint64_t recovered_subsidies = ( uint64_t( delta_time ) * uint64_t( wso.single_witness_subsidy_limit ) )
+                / (STEEM_ACCOUNT_SUBSIDY_BURST_DAYS * 60 * 60 * 24);
```

Comment:
Despite making the pool size votable is this still a reasonable heuristic to limit witness usage?